semgrep/semgrep/semgrep_main.py

```diff
@@ -68,7 +68,7 @@ def get_file_ignore() -> FileIgnore:
     workdir = Path.cwd()
     semgrepignore_path = Path(workdir / IGNORE_FILE_NAME)
     if not semgrepignore_path.is_file():
-        logger.verbose("No .semgrepignore found. Using default ignores rules")
+        logger.verbose("No .semgrepignore found. Using default .semgrepignore rules. See the docs for the list of default rules: https://semgrep.dev/docs/cli-usage/#ignoring-files")
```

Comment:
```suggestion
        logger.verbose("No .semgrepignore found. Using default .semgrepignore rules. See the docs for the list of default ignores: https://semgrep.dev/docs/cli-usage/#ignoring-files")
```

Not important, "rules" just has a specific meaning to me.